php/commands/site.php

```diff
@@ -389,10 +389,10 @@ public function backup($args, $assoc_args) {
        case 'get':
          // prompt for backup type
          if (!$element = @$assoc_args['element']) {
-           $element = Terminus::menu(array('code','files','database'), null, "Select type backup", TRUE);
+           $element = Terminus::menu(array('code','files','db'), null, "Select type backup", TRUE);
```

Comment:
Also, should we allow 'database' as a valid input here so as not to break existing scripts?